### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by dh-ecc**
Move over contents from https://github.com/dh-ecc/libzcashlc

### CODE REVIEW

File: libzcashlc.podspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    s.name             = 'libzcashlc'
+    s.version          = '0.0.1'
+    s.summary          = 'Rust core for ZCash clients'
+    s.homepage         = 'https://github.com/dh-ecc/libzcashlc'
+    s.license          = { :type => 'MIT', :file => 'LICENSE' }
+    s.author           = { 
+        'Francisco Gindre' => 'francisco.gindre@gmail.com',
+        'Jack Grigg' => 'str4d@electriccoin.co'
+     }
+    s.source           = { :git => 'https://github.com/dh-ecc/libzcashlc', :tag => s.version.to_s }
```
Comment (pacu) [inline libzcashlc.podspec:11]: shouldn't this have the url for this repo?

File: rust/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+[lib]
+name = "zcashlc"
+path = "src/lib.rs"
+crate-type = ["staticlib"]
+
+[profile.release]
+lto = true
+
+[patch.crates-io]
+zcash_client_backend = {git = "https://github.com/gmale/librustzcash", branch = "autoshield-poc-daa-taddr3"}
```
Comment (pacu) [inline rust/Cargo.toml:49]: The latest code is patched to a different repo and branch. See https://github.com/zcash/ZcashLightClientKit/blob/0.12.0-beta.6/Cargo.toml

````Toml
[patch.crates-io]
zcash_client_backend = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
zcash_client_sqlite = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
zcash_primitives = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
zcash_proofs = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
````

**Review by pacu — CHANGES_REQUESTED** [review]

File: libzcashlc.podspec (hunk 1 above)
Comment (dh-ecc) [reply libzcashlc.podspec:11]: great spot! 

Yes it should! I managed to revert this somehow :(

File: rust/Cargo.toml (hunk 2 above)
Comment (dh-ecc) [reply rust/Cargo.toml:49]: 👍 will update

Comment (dh-ecc) [reply rust/Cargo.toml]: 👍 done in [1839606](https://github.com/zcash-hackworks/zcash-light-client-ffi/pull/1/commits/1839606a4b4219e9c10cb0df94d72765fbbf049f)

**Review by pacu — APPROVED** [review]

File: libzcashlc.podspec (hunk 1 above)
Comment (dh-ecc) [reply libzcashlc.podspec]: 👍 done in [39ffd1a](https://github.com/zcash-hackworks/zcash-light-client-ffi/pull/1/commits/39ffd1a7ca87e958e7911e1c3aec5bc90c78c664)

**Pull request merged**